README.md

```diff
@@ -1,8 +1,32 @@
 # Rijks Museum Service
 
+## Overview
 This program uses the Rijks Service API to access data from the Rijks Museum.
-It shows information about different paintings in the museum.
-(Updates to follow as work continues.)
+It shows different paintings in the museum with information about each paiting.
+
+## GUI
+The gui shows 10 images at a time. 
+
+![10 Images](src/main/resources/Screenshots/RijksGUIOverview.png)
```

Comment:
While this works, you don't want you screenshots actually IN you resources folder because they aren't really part of your application. 
They can be in a folder called `Screenshots` at the root of your project.